mlrun/model_monitoring/stream_processing_fs.py

```diff
@@ -54,6 +54,7 @@
 METRICS = "metrics"
 BATCH_TIMESTAMP = "batch_timestamp"
 TIME_FORMAT: str = "%Y-%m-%d %H:%M:%S.%f"  # ISO 8061
+PROJECT = "project"
```

Comment:
PROJECT is not used, is it possible that those params will be used outside of this module? I don't see a reason not to move it to an Enum class.